k8s/raftkeeper-k8s-deployment.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
```

Comment:
File name can rename to k8s-deployment.yaml

Reply:
Done in most recent commit.